techniques/failures/F50.html

```diff
@@ -14,13 +14,13 @@
   // blink "on" state
   function show(){
       document.getElementById("blink1").style.visibility = "visible";
-      settime-out("hide()", 450);
+      window.setTimeout("hide()", 450);
```

Comment:
While we're at it, how about switching to the recommended usage, passing the function by reference rather than essentially using eval

```suggestion
      window.setTimeout(hide, 450);
```